ph5/clients/tests/test_ph5availability.py

```diff
@@ -1881,6 +1881,22 @@ def test_availability_error(self):
         self.ph5_sr_error.close()
 
 
+class TestAvailability_DateRange(LogTestCase, TempDirTestCase):
+    def test_CompareMS_Availability(self):
```

Comment:
If it is possible, please give some explanation what a unittest is trying to check or may be refer to the issue number. When the code is growing, the name of the test seem not giving enough information anymore.